examples/topology_time_series.ipynb

```diff
@@ -18,6 +18,7 @@
     "\n",
     "## See also\n",
     "\n",
+    "- [Gravitational waves detection](https://giotto-ai.github.io/gtda-docs/latest/notebooks/gravitational_waves_detection.html), where the *Takens embedding* technique helps us find recurrent structures in noisy signals from gravitational waves, and detect their presence.\n",
```

Comment:
Similar to above, e.g. `where, following [arXiv:1910.08245](https://arxiv.org/abs/1910.08245), the *Takens embedding technique* that we will present here is shown to be effective for the detection of gravitational waves signals buried in background noise.`